src/app/components/cohort-view/cohort-view.directive.js

```diff
@@ -97,6 +99,27 @@ angular.module('transmartBaseUi')
                     });
                 };
 
+
+                ctrl.loadCohort = function(cohort) {
+
+                    // Get the controller for the selected workspace
+                    var cohortSelectionCtrl = CohortSelectionService.boxes[0].ctrl;
+
+                    // Parse query XML
+                    QueryParserService.convertCohortFiltersFromXML(cohort.queryXML, cohortSelectionCtrl);
+
+                    //TODO: automatically switch to cohort selection tab
+                };
+
+                ctrl.loadSelectedCohorts = function() {
```

Comment:
I find the naming of the functions loadCohort, loadCohorts and loadSelectedCohorts are a bit confusing, add comments? or change the names?